src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormAclServiceImpl.java

```diff
@@ -35,18 +46,26 @@ public class FormAclServiceImpl implements FormAclService {
     private final ObjectMapper objectMapper;
     private final GlobalMessageSource globalMessageSource;
     private final IdGeneratorImpl idGenerator;
+    private final UserDetails userDetails;
     @Override
     public FormAclDto saveFormAcl(FormAclDto formAclDto) throws JsonProcessingException {
+        Map<String,Object> loggedInUserDetails =userDetails.getUserDetails().get(0);
```

Comment:
add this code under the if condition, i.e. only when a logged in user details is required. Also use the AuditorAware interface method to retrieve the detail